### PR TITLE
change the sqlState len

### LINKED ISSUE #8179: Error message if circuit break mode has unnecessary test.
When execute sql in circuit break mode, it display:

```
ERROR 10000 (C1000): 0Circuit break mode is ON.
```

In word `0Circuit`, `0` is unnecessary, please remove it.

image

### PR DESCRIPTION
**Pull request by totalo**
Fixes #8179 .

Changes proposed in this pull request:
- change the sqlState len

### CODE REVIEW

**Review by terrymanu — CHANGES_REQUESTED** [review]
Please fix CI

File: shardingsphere-db-protocol/shardingsphere-db-protocol-core/src/main/java/org/apache/shardingsphere/db/protocol/error/CommonErrorCode.java [hunk 1]
```diff
@@ -27,35 +27,35 @@
...
+    SHARDING_RULE_NOT_EXIST(1109, "C1109", "Sharding rule does not exist."),
     
-    SHARDING_TABLE_RULE_EXIST(11010, "C11010", "Sharding table rules: [%s] already exists."),
+    SHARDING_TABLE_RULE_EXIST(1110, "C1110", "Sharding table rules: [%s] already exists."),
     
-    SCALING_JOB_NOT_EXIST(12001, "C12001", "Scaling job %s does not exist."),
+    SCALING_JOB_NOT_EXIST(1201, "C1201", "Scaling job %s does not exist."),
     
-    SCALING_OPERATE_FAILED(12009, "C12009", "Scaling Operate Failed: [%s]"),
+    SCALING_OPERATE_FAILED(12009, "C1209", "Scaling Operate Failed: [%s]"),
```
Comment (tuohai666) [inline shardingsphere-db-protocol/shardingsphere-db-protocol-core/src/main/java/org/apache/shardingsphere/db/protocol/error/CommonErrorCode.java:54]: Do you mean 1209?

**Review by tuohai666 — CHANGES_REQUESTED** [review]

File: shardingsphere-db-protocol/shardingsphere-db-protocol-core/src/main/java/org/apache/shardingsphere/db/protocol/error/CommonErrorCode.java (hunk 1 above)
Comment (totalo) [reply shardingsphere-db-protocol/shardingsphere-db-protocol-core/src/main/java/org/apache/shardingsphere/db/protocol/error/CommonErrorCode.java:54]: Yes

**Comment by tuohai666** [comment]
@totalo The ci still failed. Please check the test cases.

**Comment by totalo** [comment]
> @totalo The ci still failed. Please check the test cases.

done , thx

**Review by tuohai666 — APPROVED** [review]

**Review by terrymanu — APPROVED** [review]

**Pull request merged**